package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "weex-playground",
+  "version": "0.0.1",
+  "description": "Playground for Apache Weex, demo purpose only",
+  "scripts": {
+    "danger": "danger"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/apache/incubator-weex-playground.git"
+  },
+  "keywords": [
+    "weex",
+    "playground"
+  ],
+  "license": "ISC",
```

Comment:
This should be `Apache-2.0`

Reply:
The license has been corrected, thanks for reviewing!

Reply:
> This should be `Apache-2.0`

The license has been corrected, thanks for reviewing!

Reply:
The license has been corrected, thanks for reviewing!